schedulearn-frontend/src/components/NestedList/TopicAddModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Modal, { setAppElement } from "react-modal";
+import { TopicAddForm, TopicForm } from "./TopicAddForm";
+import "./TopicAddModal.scss";
+
+setAppElement("#root");
+
+export interface TopicAddModalProps {
+  isOpen: boolean;
+  disabled: boolean;
+  topic?: TopicForm;
+  onRequestClose: () => void;
+  onEventSubmit: (topic: TopicForm) => void;
+}
+
+export class TopicAddModal extends React.PureComponent<TopicAddModalProps> {
+  render(): JSX.Element {
+    return (
+      <Modal
+        isOpen={this.props.isOpen}
```

Comment:
Needs to set portalClassName to ".ReactModal__Portal topic-add-modal" so the style could be applied specifically for your ReactModal. Check andrius Info modal.

Reply:
`portalClassName` set to `topic-add-modal ReactModalPortal`